lib/sqlalchemy/engine/url.py

```diff
@@ -755,6 +755,13 @@ def _parse_rfc1738_args(name):
             query = None
         components["query"] = query
 
+        if components["query"]:
```

Comment:
**mike bayer** (zzzeek) wrote:

OK, this is the idea but this should be done in psycopg2.py -> create_connect_args().

it also requires a doc section in psycopg2.py -> probably right after "Unix Domain Connections"

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2206